addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -38,11 +38,10 @@
         <t t-if="message.isSelfAuthored">
             <i class="fa fa-mail-reply me-1"/>You:
         </t>
-        <t t-elif="message.author and !message.author?.eq(thread.correspondent?.persona)">
-            <t t-esc="message.author.name"/>:
+        <t t-elif="message.author and thread.correspondent and message.author?.eq(thread.correspondent.persona)">
         </t>
-        <t t-elif="message.email_from">
-            <t t-esc="message.email_from"/>:
+        <t t-elif="!thread.correspondent">
+            <t t-esc="message.author?.name ?? message.email_from ?? ''"/>:
```

Comment:
`t-esc`ing undefined or null values does not output anything to the DOM while empty string does, last nullish coalescing can be omitted.